src/yashCJun20/Assignment22/VerifyPalindromeNumber.java

```diff
@@ -0,0 +1,25 @@
+package yashCJun20.Assignment22;
+class VerifyPalindromeNumber{
+	void toVerifyPalindromeNumberOrNot(int num){
+		int remender;
```

Comment:
No need of this variable. Before use any local variable you should initialise it.